src/test/end-to-end/shared/shared-functions.ts

```diff
@@ -1,3 +1,63 @@
+import fs from 'fs';
+import { randomData } from './random-data';
+import path from 'path/posix';
+import os from 'os';
+
+const mockLocationFilePrefix = 'mock-location-';
+
+const locationJsonData = (id, name) => {
+    return [
+        {
+            'P&I ID': id,
+            'Court Desc': name,
+            'Welsh Court Desc': name,
+            Region: 'South East',
+            'Welsh Region': 'De-ddwyrain Lloegr',
+            Jurisdiction: 'Family; Civil',
+            'Welsh Jurisdiction': 'Llys Sifil; Llys Teulu',
+            Provenance: 'ListAssist',
+            'Provenance Location ID': '3482',
+            'Provenance Location Type': 'Venue',
+        },
+    ];
+};
+
+const generateTestLocationFields = (): [number, string] => {
+    // Generate a random test court ID between 1000 and 9999999999
+    const id = randomData.getRandomNumber(1000, 9999999999);
```

Comment:
The E2E test seems to be failing (ran a few times with same error). 

I think (but not checked the logs) it could be to do with the max int here. It's above the max java int (2147483647) so I think it's worth changing it to that and trying again

Reply:
Have removed one digit from the max number. Doesnt need to be that huge. It is passing now